python2/raygun4py/raygunmsgs.py

```diff
@@ -128,14 +129,16 @@ def get_details(self):
 
 class RaygunErrorMessage(object):
 
+    INSPECT_FRAME_TYPE_SIGNATURE = r'<frame [\w ]*0x\w*[>,]'
```

Comment:
Seems like this regex works relatively well for both Python 2 & 3.

https://regexr.com/

![Screen Shot 2019-09-04 at 5 49 45 PM](https://user-images.githubusercontent.com/14981893/64298406-6bcf0a80-cf43-11e9-8b20-c0e6415a63cd.png)